language-server/src/server.js

```diff
@@ -367,5 +372,25 @@ connection.languages.semanticTokens.onDelta(async ({ textDocument, previousResul
   return builder.buildEdits();
 });
 
+// $SCHEMA COMPLETION
+connection.onCompletion((textDocumentPosition) => {
+  const doc = documents.get(textDocumentPosition.textDocument.uri);
+  if (!doc) {
+    return [];
+  }
+
+  const instance = JsoncInstance.fromTextDocument(doc);
+
+  const currentProperty = instance.getInstanceAtPosition(textDocumentPosition.position);
+  if (currentProperty.pointer.endsWith("/$schema")) {
+    return getDialectIds().map((uri) => {
+      return {
+        label: `"${uri}"`,
```

Comment:
This line needs to be reverted as well. We don't want the quotes.

```suggestion
        label: uri,
```